wolfssl/wolfcrypt/asn_public.h

```diff
@@ -250,7 +250,12 @@ enum {
 #else
     NAME_SZ = 80,                   /* max one line */
 #endif
-
+#if defined(WOLFSSL_CERT_EXT) || defined(OPENSSL_EXTRA) || \
+        defined(HAVE_PKCS7) || defined(OPENSSL_EXTRA_X509_SMALL) || \
+        defined(HAVE_OID_DECODING) || defined(HAVE_OID_ENCODING)
+    MAX_OID_SZ          = 32,      /* Max DER length of OID*/
+    MAX_OID_STRING_SZ   = 64,      /* Max string length representation of OID*/
```

Comment:
If we are going to put a size constraint on it. Please make `MAX_OID_STRING_SZ` and build-time overridable Marco. If it needs to be moved to a `#define` it is fine with me.

Reply:
Will do...stay tuned!